### ISSUE TITLE
Builds fail on macOS

### ISSUE THREAD

**Issue by revolter** [Chatterino/chatterino2#2383]
**Describe the bug**

Related to #1328, which fixes it for Travis CI, but not for local builds too:

> I think we should merge this until we can figure out what's actually wrong.

**To reproduce**

https://github.com/Chatterino/chatterino2/blob/master/BUILDING_ON_MAC.md

**Screenshots**

**Chatterino version**

**Operating system**

macOS Big Sur 11.1 (20C69)

**Additional information**

**Comment by leon-richardt** [comment]
Could you provide the error message you're getting?

**Comment by revolter** [comment]
The same as on Travis:
```
clang: error: unknown argument: '-framework Security'
clang: error: unknown argument: '-framework Foundation'
```

**Comment by revolter** [comment]
As a workaround, manually replacing `-framework\ Security -framework\ Foundation` with `-framework Security -framework Foundation` in `Makefile` and then running only `make`, without `qmake ..` works.

**Comment by revolter** [comment]
After https://github.com/Chatterino/qtkeychain/pull/1 gets merged, #1328 should be reverted.

**Comment by fanway** [comment]
Github actions should be fixed too :)
https://github.com/Chatterino/chatterino2/blob/857705668e65bceeb9833ff87f61b3ac11245bda/.github/workflows/build.yml#L162

**Comment by pajlada** [comment]
Fixed in #2393
